lumicks/pylake/tests/test_file.py

```diff
@@ -76,6 +76,25 @@ def test_properties(h5_file):
     assert f.fdcurves == {}
 
 
+def test_groups(h5_file):
+    f = pylake.File.from_h5py(h5_file)
+    if f.format_version == 1:
```

Comment:
Not related to this PR, but just a note for the future. We should start thinking about a better way to do file fixture versioning since the conditions are going to get out of hand.

Reply:
Yep. Noted.